meta-lmp-base/recipes-core/ovmf/ovmf/0001-CryptoPkg-Increase-ScratchMemory-buffer-for-openssl-.patch

```diff
@@ -0,0 +1,41 @@
+From 46e407d9654d5c8986b0f7b61de3aa223b1df399 Mon Sep 17 00:00:00 2001
+From: Jorge Ramirez-Ortiz <jorge@foundries.io>
+Date: Thu, 31 Oct 2024 15:12:04 +0100
+Subject: [PATCH] CryptoPkg: Increase ScratchMemory buffer for openssl 3.0.15
+
+Openssl 3.0.15 has a larger memory footprint.
+
+Updating from EDK 2022.2 (openssl 1.1.j) to 2024.2 (openssl 3.0.15)
+causes our EFI provisioning application[1] to fail due to an out of
+memory condition.
+
+On inspection, at the time of that fault, 2022.2 had an additional 900
+pages. This is why this patch proposes the increase of the ScratchMemory
+buffer by that same ammount.
+
+[1] https://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git
+
+Upstream-Status: Submitted [https://github.com/tianocore/edk2/pull/6394]
```

Comment:
The upstream status should be Backport as the path is already merged

Reply:
sure, did just replicate the EDK2 so we should then just change both?

Reply:
right, so we can update the both to be Backport.